src/stylesheets/elements/_inputs.scss

```diff
@@ -145,16 +145,16 @@ input[type="radio"] + label::before {
   content: '\a0';
   display: inline-block;
   height: 1.8rem;
-  line-height: 0.8;
+  line-height: 1.8rem;
```

Comment:
Yes, it accounts for that. This only looks at the size of the button not text, it will always be centered.

Here I made the text way bigger and it's still centered:
<img width="338" alt="screen shot 2016-04-19 at 11 25 23 am" src="https://cloud.githubusercontent.com/assets/5249443/14650650/7ded2542-0621-11e6-96c1-136293dddc2d.png">